src/t_zset.c

```diff
@@ -424,7 +424,7 @@ unsigned long zslDeleteRangeByLex(zskiplist *zsl, zlexrangespec *range, dict *di
         update[i] = x;
     }
 
-    /* Current node is the last with score < or <= min. */
+    /* Current node is the last with lexicographic value < or <= min. */
```

Comment:
in this context, a "lexicographic value" and a "score" have the same meaning.
i.e. the sorted-set score is the lexicographic value.
why bother to change these? am i missing something?

Reply:
I just think "lexicographic value" means more accurate...
I think sorted-set's real score is not the "lexicographic value",it used `ele` to compare instead of `score` .

Reply:
we could also maybe settle for `lexicographic score`. but one of the things that bother me is that i'm not sure all these lines should be changed at all (trying to resist a change for the sake of change).

Reply:
Well, I think the meaning of this "score" can be easily understood from the context.
So I agree with you that there is no need to change it.